test/ws-api/scenarios/multiple-clients.test.ts

```diff
@@ -0,0 +1,52 @@
+import * as path from 'path'
+import { captureConsole } from '../../support/captureConsole'
+import { runBrowserWith } from '../../support/runBrowserWith'
+
+test('sends server event to all active clients', async () => {
```

Comment:
this test is great! I'd like to try it out in my tests, thanks for all this hard work here ✨

Reply:
Thanks, @keremciu. So far it behaves as expected in regards to multi-client handling. Please let me know once you try this out.